open-source-license-acknowledgements-and-third-party-copyrights.md

```diff
@@ -330,7 +330,6 @@ You can check all the dependencies using the instructions from the section Libra
 | @babel/preset-env                         | MIT                                                    | The Babel Team (https://babel.dev/team)                                                              | https://babel.dev/docs/en/next/babel-preset-env                         | https://github.com/babel/babel.git                                                            |
 | @babel/traverse                           | MIT                                                    | The Babel Team (https://babel.dev/team)                                                              | https://babel.dev/docs/en/next/babel-traverse                           | https://github.com/babel/babel.git                                                            |
 | @symfony/webpack-encore                   | MIT                                                    | Ryan Weaver <ryan@knpuniversity.com>                                                                 | https://github.com/symfony/webpack-encore                               | git+https://github.com/symfony/webpack-encore.git                                             |
-| @vusion/webfonts-generator                | Unlicense                                              | moujintao <moujintaotest@163.com>                                                                    | -                                                                       | git+https://github.com/vusion/webfonts-generator.git                                          |
```

Comment:
Shouldn't we update previous section and write there about that newly installed composer package?

Reply:
it is ensured automatically during the release process (see `UpdateLicenseAcknowledgementsReleaseWorker`)